handler/panos.go

```diff
@@ -19,6 +21,9 @@ const (
 	// Users with custom roles currently return an error for an empty commit with
 	// this server response prefix. See GH-73 for more details.
 	emptyCommitServerRespPrefix = `<response status="success" code="13">`
+
+	// max number of retries
+	maxRetris = 4
```

Comment:
small typo: "maxRetris" => "maxRetries"

Reply:
Saw the approved and missed this. Will fix in a small follow up commit.